measures/TableMeasures/ScalarQuantColumn.h

```diff
@@ -194,6 +194,14 @@ template<class T> class ScalarQuantColumn
   // Throw an exception if the object is null.
   void throwIfNull() const;
 
+  // get the column as a Quantum<Vector<T> >. If the units are variable, the
+  // values in the returned Vector have been converted to the
+  // unit of the 0th row entry. Otherwise, the units of the returned
+  // Quantum are itsUnit (no conversion to itsUnitOut is attempted if
+  // itsConvOut is True, because of compile issues if the column type
+  // is Complex).
```

Comment:
what compile issue exactly?
I think an optional unit argument to this function would be useful